xdsl/transforms/printf_to_llvm.py

```diff
@@ -135,9 +135,7 @@ def match_and_rewrite(self, op: printf.PrintFormatOp, rewriter: PatternRewriter,
         rewriter.replace_matched_op(
             casts
             + [
-                ptr := llvm.AddressOfOp.get(
-                    globl.sym_name, llvm.LLVMPointerType.opaque()
-                ),
+                ptr := llvm.AddressOfOp(globl.sym_name, llvm.LLVMPointerType.opaque()),
```

Comment:
I guess this warning was just ignored? :thinking: 
Should we check that this does not happen?

Reply:
Yeah it was just ignored, I'm not sure exactly what the deal is, I think deprecated doesn't work well with static methods.

Reply:
Good thing we have used it for all the constructors xD

Reply:
:)